src/materialized/src/bin/materialized/main.rs

```diff
@@ -493,6 +494,18 @@ fn run(args: Args) -> Result<(), anyhow::Error> {
     // When inside a cgroup with a cpu limit,
     // the logical cpus can be lower than the physical cpus.
     let ncpus_useful = usize::max(1, cmp::min(num_cpus::get(), num_cpus::get_physical()));
+    let memory_limit = detect_memory_limit().unwrap_or_else(|| MemoryLimit {
```

Comment:
would you mind also adding these values to the prometheus "system stats" labels? 

https://github.com/MaterializeInc/materialize/blob/67017c7dce82536507183e990ae327e91b0e4502/src/materialized/src/lib.rs#L211-L225

Reply:
Sure!

Reply:
What do we want if there is no limit? Just a string `"None"`?

Reply:
Added.